dss-api.yml

```diff
@@ -258,6 +258,79 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    put:
+      summary: Create a bundle
+      description: |
+        Create a new bundle with a given UUID.  The list of files UUID+versions to be included must be provided.
+      parameters:
+        - name: uuid
+          in: path
+          description: A RFC4122-compliant ID for the file.
+          required: true
+          type: string
+          pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+        - name: version
+          in: query
+          description: Timestamp of bundle creation in RFC3339.
+          required: false
+          type: string
+          format: date-time
+        - name: replica
+          in: query
+          description: Replica to write to.
+          required: true
+          type: string
+          pattern: "^(AWS|GC|Azure)$"
```

Comment:
Does the spec say how this should be formatted? I was using `(aws|gcs|azure)` elsewhere, we should make sure they are consistent.

Also, I believe swagger has enums, up to you if you want to use it.